home/tests/test_content_import_export.py

```diff
@@ -1882,3 +1884,128 @@ def test_list_items(self, impexp: ImportExport) -> None:
         impexp.export_reimport()
         imported = impexp.get_page_json()
         assert imported == orig
+
+    def test_content_page_with_no_go_to_button(self, impexp: ImportExport) -> None:
+        """
+        If a page has a button go to page it should export all buttons.
```

Comment:
The test name and docstring don't seem to agree, and I'm not really sure what this is intended to test. Can you clarify the docstring and make sure the test name matches, please?